types/react/experimental.d.ts

```diff
@@ -238,4 +240,35 @@ declare module "." {
     interface DO_NOT_USE_OR_YOU_WILL_BE_FIRED_EXPERIMENTAL_MEDIA_SRC_TYPES {
         srcObject: Blob | MediaSource | MediaStream;
     }
+
+    // @enableStore
+    /**
+     * The return value of `createStore`.
+     */
+    // Value is unused in the shape because it is a private property that's only
+    // accessible via `use(store)`.
+    interface Store<Value, Action> {
```

Comment:
```suggestion
    interface Store<out Value, in Action> {
```

Reply:
I haven't touched these perf hints. I don't know how you would measure it or validate that they're actually correct.

Reply:
They're not for perf, but they define behaviour. Typescript makes everything co-variant by default (`out` in this case), so `out` is generally optional, but `Action` is something that goes _into_ the store and needs an `in` annotation to work correctly.